### DISCUSSION TITLE
DQ点单小程序 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#182] [模板讨论]
DQ点单小程序 模板问题反馈评论区
有问题可以下面反馈。

**Comment by zadishu** [comment]
显示成功，实际不成功

**Reply by wjf0214** [reply]
我目前使用正常。如果确定DQ正常绑定的手机号不能用的话，方便私发我手机号测试下。

**Comment by zadishu** [comment]
Failed at 3/5 request,
Fail assert: {"re": "error", "from": "content"} from failed_asserts,
Request URL: https://wechat.dairyqueen.com.cn/loginNoLandfall

**Reply by monobo1989** [reply]
我也这样，怎么办啊？？？
废了吗？

**Comment by monobo1989** [comment]
2024-03-22 23:05:45
日志：Failed at 3/5 request,
Fail assert: {"re": "error", "from": "content"} from failed_asserts,
Request URL: https://wechat.dairyqueen.com.cn/loginNoLandfall

**Reply by monobo1989** [reply]
没法用了，咋办？2024-03-22 23:05:45
日志：Failed at 3/5 request,
Fail assert: {"re": "error", "from": "content"} from failed_asserts,
Request URL: https://wechat.dairyqueen.com.cn/loginNoLandfall

**Comment by miaoxiaozhao** [comment]
2024-03-25 12:03:13
日志：Failed at 3/5 request,
Fail assert: {"re": "error", "from": "content"} from failed_asserts,
Request URL: https://wechat.dairyqueen.com.cn/loginNoLandfall

**Comment by wjf0214** [comment]
@zadishu @monobo1989 @17huiwei 已更新。增加cookie和ua需要抓包获取。

**Reply by miaoxiaozhao** [reply]
测试成功

**Reply by monobo1989** [reply]
可以用了。CK有效期多久呀？不会一天就掉吧？

**Reply by zadishu** [reply]
抓取的cookie，半天就失效了，
JSESSIONID=c77d7651-6f4c-477a-94b3-8847fdc3b6b8

**Reply by monobo1989** [reply]
那搞屁啊。相当于还是废了

**Reply by wjf0214** [reply]
试了下。的确很快就失效了，有空再看看能不能延长有效期或者重新获取吧。

**Comment by windy1021** [comment]
麻烦有新版了踢我一下，感谢

**Comment by monobo1989** [comment]
麻烦有新版了踢我一下，感谢

**Reply by monobo1989** [reply]
看来这个羊毛黄了啊，呜呜

**Comment by monobo1989** [comment]
看来这个羊毛黄了哎
